mmv1/templates/terraform/examples/dlp_job_trigger_publish_to_stackdriver.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_data_loss_prevention_job_trigger" "<%= ctx[:primary_resource_id] %>" {
+  parent       = "projects/<%= ctx[:test_env_vars]['project'] %>"
+  description  = "Description for the job_trigger created by terraform"
+  display_name = "TerraformDisplayName"
+
+  triggers {
+    schedule {
+      recurrence_period_duration = "86400s"
+    }
+  }
+
+  inspect_job {
+    inspect_template_name = "sample-inspect-template"
+    actions {
+      publish_to_stackdriver {}
```

Comment:
Could you also test another non-empty value for this field? Thanks.

Reply:
It seems that `publishToStackdriver` field is [an empty object that acts as a boolean](https://github.com/hashicorp/terraform-provider-google/issues/13768#issuecomment-1438887160). I've confirmed this behavior using Terraform and API explorer as well. The behavior of this field is similar to other actions such as `jobNotificationEmails`, `publishFindingsToCloudDataCatalog`, and `publishSummaryToCscc`. The API reference for this field can be found [here](https://cloud.google.com/dlp/docs/reference/rest/v2/Action#PublishToStackdriver).